### PR TITLE
resource group api

### PR DESCRIPTION
**Pull request by weiliu1031**
Signed-off-by: Wei Liu <wei.liu@zilliz.com>
Related to: #76

### CODE REVIEW

File: proto/common.proto [hunk 1]
```diff
@@ -243,6 +243,13 @@ enum MsgType {
     SelectGrant = 1607;
     RefreshPolicyInfoCache = 1608;
     ListPolicy = 1609;
+
+    /* Resource group */
+    CreateResourceGroup = 1700;
+    DropResourceGroup = 1701;
+    ListRGroupesource = 1702;
+    DescribeResourceGroup = 1703;
+    TransferNode = 1704;
```
Comment (xiaofan-luan) [inline proto/common.proto:252]: TransferCollection

File: go-api/milvuspb/milvus.pb.go [hunk 2]
```diff
@@ -7759,6 +7777,461 @@ func (m *CheckHealthResponse) GetQuotaStates() []QuotaState {
...
+}
+
+func (m *ReplicaNodesInfo) GetOutboundNodes() []int64 {
+	if m != nil {
+		return m.OutboundNodes
+	}
+	return nil
+}
+
+type ResourceGroup struct {
```
Comment (xiaofan-luan) [inline go-api/milvuspb/milvus.pb.go:7835]: Maybe also need to find all collection/replica info in a group

File: proto/milvus.proto [hunk 3]
```diff
@@ -1391,3 +1404,62 @@ message CheckHealthResponse {
...
+  int64  replicaID = 3;
+}
+
+message ListResourceGroupRequest {
+  common.MsgBase base = 1;
+}
+
+message ListResourceGroupResponse {
+  common.Status status = 1;
+  repeated string  resouse_groups = 2;
```
Comment (czs007) [inline proto/milvus.proto:1454]: typo

File: proto/milvus.proto [hunk 4]
```diff
@@ -1391,3 +1404,62 @@ message CheckHealthResponse {
...
+}
+
+message DescribeResourceGroupRequest {
+  common.MsgBase base = 1;
+  string resource_group = 2;
+}
+
+message DescribeResourceGroupResponse {
+  common.Status status = 1;
+  ResourceGroup  resouse_group = 2;
```
Comment (czs007) [inline proto/milvus.proto:1464]: typo

File: proto/common.proto [hunk 5]
```diff
@@ -243,6 +243,16 @@ enum MsgType {
     SelectGrant = 1607;
     RefreshPolicyInfoCache = 1608;
     ListPolicy = 1609;
+
+    /* Resource group */
+    CreateResourceGroup = 1700;
+    DropResourceGroup = 1701;
+    ListRGroupesource = 1702;
```
Comment (czs007) [inline proto/common.proto:250]: typo

**Pull request merged**